packages/asc-ui/src/components/Icon/IconStyle.ts

```diff
@@ -1,10 +1,8 @@
 import styled, { css } from 'styled-components'
```

Comment:
Could you also add the breaking changes to the changelog @jasperswart?

Reply:
yes I will do that